gemrb/vcpkg_deps.cmake

```diff
@@ -66,6 +66,19 @@ FOREACH(ENTRY IN LISTS DLL_SET_RELEASE)
 	ENDIF()
 ENDFOREACH()
 
+# a custom target which copies the dll files to the build directory if needed, useful for rapid testing
+IF(${CMAKE_BUILD_TYPE} STREQUAL "Debug")
+	ADD_CUSTOM_COMMAND(TARGET gemrb POST_BUILD
+		COMMAND ${CMAKE_COMMAND} -E copy_if_different
+		${DLL_PATHS_DEBUG}
+		${CMAKE_BINARY_DIR}/gemrb/)
+ELSE()
+	ADD_CUSTOM_COMMAND(TARGET gemrb POST_BUILD
+		COMMAND ${CMAKE_COMMAND} -E copy_if_different
+		${DLL_PATHS_RELEASE}
+		${CMAKE_BINARY_DIR}/gemrb/)
+ENDIF()
+
```

Comment:
I think it would be cleaner with a bit less duplication. My suggestion:
- split the if into determining which path to use
- then unconditionally add the custom command with the new var